apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.tsx

```diff
@@ -64,7 +64,8 @@ export const BrowsePools = ({ pools }: BrowsePoolsProps) => {
   const activeRecruitmentPools = pools.filter(
     (p) =>
       p.status === PoolStatus.Published && // list jobs which have the PUBLISHED PoolStatus
-      p.publishingGroup === PublishingGroup.ItJobs, // and which are meant to be published on the IT Jobs page
+      (p.publishingGroup === PublishingGroup.ItJobs ||
+        p.publishingGroup === PublishingGroup.ExecutiveJobs), // and which are meant to be published on the IT Jobs page
```

Comment:
I don't think this comment makes sense: there is no IT Jobs page.
```suggestion
        p.publishingGroup === PublishingGroup.ExecutiveJobs),
```